assets/src/edit-story/components/panels/utils/getBoundRect.js

```diff
@@ -0,0 +1,149 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import { dataPixels } from '../../../units/dimensions';
+
+/**
+ * Get the bound rect value for all objects in `list`,
+ *
+ * ```
```

Comment:
?

Reply:
Get the outer frame of the objects in list. That includes rotated object.

Reply:
@ndev1991 Sorry, I should have been clearer. These are some random back ticks here. Might wanna remove them, unless you intended to add an example here.

Reply:
Okay

Reply:
Removed! 😄